### PR TITLE
Jasmine setup

### PR DESCRIPTION
**Pull request by nixlim**

### CODE REVIEW

File: server/controllers/app_tester.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const express = require('express')
```
Comment (williampowell92) [inline server/controllers/app_tester.js]: Delete this file

File: test/exampleForTestingSpec.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('A Calc Test', function () {
```
Comment (williampowell92) [inline test/exampleForTestingSpec.js]: Delete this file

**Review by williampowell92 — CHANGES_REQUESTED** [review]

**Review by nixlim — COMMENTED** [review]
done

**Review by williampowell92 — APPROVED** [review]

**Comment by williampowell92** [comment]
:shipit:

**Pull request merged**